pkg/runtime/wfengine/activity.go

```diff
@@ -104,7 +105,7 @@ func (a *activityActor) InvokeMethod(ctx context.Context, actorID string, method
 
 // InvokeReminder implements actors.InternalActor and executes the activity logic.
 func (a *activityActor) InvokeReminder(ctx context.Context, actorID string, reminderName string, data []byte, dueTime string, period string) error {
-	wfLogger.Debugf("invoking reminder '%s' on activity actor '%s'", reminderName, actorID)
+	wfLogger.Debugf("Invoking reminder '%s' on activity actor '%s'", reminderName, actorID)
```

Comment:
Curious about this change - I've recently been seeing the VS Code linter for Go give me warnings about starting logs (or maybe it was just error messages?) with capital letters - i.e., it wanted me to start them with lowercase. Do we have separate guidance for capitalization in the Dapr project?

Reply:
Error messages always start lowercase per Go convention: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

For logs, we generally try to make them start with an uppercase letter.